cockpit/leapp.html

```diff
@@ -803,63 +866,65 @@ <h3 class="pull-left">
         }
 
         function load_target_container_names() {
-            var ctx = new_leapp_call(["check-target", "-t", "127.0.0.1"]);
+            var cmd = ["check-target", {
+                target_host: "localhost"
+            }]
+            var ctx = new_leapp_call(cmd);
             ctx.log("Loading used applications names from the target...");
             ctx.done(set_app_list);
             ctx.run();
             return ctx;
         }
 
-        function override_identity(cmd_args) {
-            // Appends any defined identity override settings to a command
-            if ("override-username" in LEAPP_CONF) {
-                cmd_args.push("--target-user", LEAPP_CONF["override-username"])
-                cmd_args.push("--source-user", LEAPP_CONF["override-username"])
-            }
-            return cmd_args;
-        }
-
         function import_to_host(event) {
+            var source = $("#source-address").val();
```

Comment:
instead of creating a lot of vars, maybe it's better to create js object and pass it to new_leapp_call